common/.eslintrc.fast.json

```diff
@@ -2,22 +2,34 @@
   "parser": "@typescript-eslint/parser",
   "plugins": [
     "@typescript-eslint",
-    "no-only-tests"
+    "no-only-tests",
+    "import"
   ],
   "extends": [
     "eslint:recommended",
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended"
   ],
   "rules": {
+    "@typescript-eslint/array-type": ["error", { "default": "array-simple", "readonly": "generic" }],
     "@typescript-eslint/ban-ts-ignore": "off",
+    "@typescript-eslint/consistent-type-definitions": "error",
```

Comment:
It's not clear from the [rule description](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/consistent-type-definitions.md) that a default type definition exists.  Can you specify explicitly `interface`?